docs/database.md

```diff
@@ -9,83 +9,90 @@ Fedimint uses a simple key-value store as its database. In theory any such KV st
 We currently use RocksDB on both the client and the server.
 
 ## Server DB Layout
-The Database is split into different key spaces based on prefixing that can be understood as different tables (each "table's" content can be retrieved using prefix search). There are three general prefix ranges:
+The database is logically partitioned based on the module instance id of the module. Each module's keyspace is split up based on prefixing. Each prefix within a module identifies a logical entity.
+Entity prefixes between modules can overlap because they are logically partitioned by the module instance id. The key value pairs that belong to consensus do not belong to any specific module, so they are
+not prepended with a module prefix. Below is the format for prefixes within a module:
+
+<GLOBAL PREFIX BYTE><2 BYTE MODULE ID><ENTITY PREFIX>
```

Comment:
What is the global prefix byte?

Reply:
0xFF. I'll update the comment so its clear.